kolibri/plugins/learn/assets/src/views/action-bar-search-box/index.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+
+  <div class="search-box-wrapper">
+    <div v-show="showSearchBox" :class="{ 'search-box-dropdown': showDropdownSearchBox }">
+      <search-box
+        ref="searchBox"
+        :icon="showDropdownSearchBox ? 'arrow_forward' : 'search'"
+        :width="showDropdownSearchBox ? 'calc(100vw - 120px)' : '150px'"
```

Comment:
`vw` and `calc` might not be supported in all versions of android - if you haven't, I would recommend testing this